app/models/user.rb

```diff
@@ -41,6 +41,12 @@ class User < ActiveRecord::Base
   before_save :set_token
   after_destroy :destroy_drupal_user
 
+  # User status
+  # 0: banned
+  # 1: active
+  # 5: moderated
+  enum status: {banned: 0, active: 1, moderated: 5}
```

Comment:
Wow, i have never used `enum` like this. Can you link me to some documentation to follow how it works?